Elastix/Elastix.py

```diff
@@ -685,6 +697,28 @@ def registerVolumes(self, fixedVolumeNode, movingVolumeNode, parameterFilenames
 
     self.addLog("Registration is completed")
 
+  def saveNodeToTemporaryLocation(self, node, filePath):
+      # Save original file paths
+      originalFilePath = ""
+      originalFilePaths = []
+      storageNode = node.GetStorageNode()
```

Comment:
It might be simpler to use the recently added `qSlicerCoreIOManager::exportNodes` methods. It is guaranteed that exporting has no side effect.

Reply:
I see. Just added a commit for this